SCClassLibrary/Common/GUI/PlusGUI/Control/server-scope.sc

```diff
@@ -24,8 +24,8 @@
 }
 
 + Bus {
-	scope { arg bufsize = 4096, zoom;
-		^server.scope(numChannels, index, bufsize, zoom, rate);
+	scope { arg bufsize = 4096, zoom, bounds = nil;
```

Comment:
no need for an explicit `nil` default value. `zoom` is also nil.

Reply:
Yes! I removed `= nil` in argument in [server-scope.sc](https://github.com/supercollider/supercollider/pull/6230/commits/25d259ab8d27686a0b478ccc8c28390337e84965) and [server-meter.sc](https://github.com/supercollider/supercollider/pull/6230/commits/e76ab825471db0e5b0d0227332ce7045d901611f)